### DISCUSSION TITLE
Images in transforms.json not sequential

### DISCUSSION THREAD

**Discussion by andyvirt** [NVlabs/instant-ngp#674] [Q&A]
Hi All,

I've created 2 videos, 10 to 15 seconds long and gone through the process several times but noticed that the image list in the transforms.json file are not in order.

When I go through the final step I get errors - one being out of memory.  The video is 1080p - 20 photos.  I then did 720p.

However, I can use the fox example just fine.

Must the data in the transforms file be sequential?  Could that be causing the errors?

**Comment by qhdqhd** [comment]
No, data in the transforms file need not to be sequential

**Reply by andyvirt** [reply]
Thanks very much qhdqhd - I can stop worrying about that and concentrate on other issues now then.
